main/src/library/Eff/Random.flix

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright 2024 Magnus Madsen
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+mod Eff.Random {
```

Comment:
I think its weird to have the `Eff` namespace, are all effects gonna go here?

Reply:
This is intended as temporary. It is just a place for us to start.